app/posts/manage-school-placements/2024-01-30-what-is-a-mentor.md

```diff
@@ -0,0 +1,69 @@
+---
+title: What is a mentor?
+description: A mentor supports trainee or early career teachers (ECTs) or oversees other mentors
+date: 2024-01-30
+tags:
+  - mentors
+related:
+  items:
+    - text: "Initial teacher training (ITT): criteria and supporting advice"
+      href: https://assets.publishing.service.gov.uk.mcas.ms/media/64ef61b813ae15000d6e30c1/Initial_teacher_training_criteria_and_supporting_advice_2024_to_2025.pdf?McasCtx=4&McasTsid=20892
+    - text: "Guidance for mentors: how to support ECF-based training"
+      href: https://www.gov.uk/guidance/guidance-for-mentors-how-to-support-ecf-based-training
+---
+
+A mentor supports trainee or early career teachers (ECTs) or oversees other mentors.
+
+Mentors need the appropriate prior experience and typically undertake training tailored to their mentor role. They can undertake training in parallel to carrying out their role.
+
+## Initial Teacher Training (ITT)
```

Comment:
lower case! 😄  See [style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style).

```suggestion
## Initial teacher training (ITT)
```